tests/api2/activedirectory.py

```diff
@@ -8,29 +8,25 @@
 from time import sleep
 apifolder = os.getcwd()
 sys.path.append(apifolder)
-from auto_config import pool_name, ip, hostname, scale, user, password
-from functions import GET, POST, PUT, DELETE, SSH_TEST, wait_on_job
+from auto_config import pool_name, ip, hostname, user, password
+from functions import (
+    GET,
+    POST,
+    PUT,
+    DELETE,
+    SSH_TEST,
+    cmd_test,
+    wait_on_job
+)
 
 try:
     from config import AD_DOMAIN, ADPASSWORD, ADUSERNAME, ADNameServer
+    ad_user = fr"AD01\{ADUSERNAME.lower()}"
```

Comment:
There is no difference between these 2 variables from what I can tell. Also since these are global variables, capitalize them.

Reply:
I had to make one `fr"AD01\\{ADUSERNAME.lower()}"` or the smbclient command fails.

Reply:
Sure but there are 2 variables that are the same? They're both using `fr'` syntax

Reply:
Oh and fr"AD01\\{ADUSERNAME.lower()}" fails the for the API call

Reply:
Oh, I see now the `cmd_ad_user` has double `\\` instead of just one.

Reply:
So `ad_user` is used for changing the perms and `cmd_ad_user` is for running the `smbclient` command.

Reply:
Yes but one contains `//`.

Reply:
> So `ad_user` is used for changing the perms and `cmd_ad_user` is for running the `smbclient` command.
Yes